barman/recovery_executor.py

```diff
@@ -1470,15 +1342,15 @@ def _backup_copy(
                     self.compression.file_extension,
                 )
                 tablespace_src_path = "%s/%s" % (staging_dir, tablespace_file)
-                output = self.compression.uncompress(
+                cmd_output = self.compression.uncompress(
                     tablespace_src_path, tablespace_dst_path
                 )
-                print(output)
+                print(cmd_output)
         base_src_path = "%s/%s" % (staging_dir, base_file)
-        output = self.compression.uncompress(
+        cmd_output = self.compression.uncompress(
             base_src_path, dest, exclude=["recovery.conf", "tablespace_map"]
         )
-        print(output)
+        print(cmd_output)
```

Comment:
I might need to remove those

Reply:
perhaps they should be info or debug level logs?

Reply:
if it was useful at dev time chances are it might be useful at debugging-in-production time 😄